ofrak_core/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to `ofrak` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+### Changed
+- Refactored `DataService` internals to more efficiently find resources affected by patches [#140](https://github.com/redballoonsecurity/ofrak/pull/140)
+- Deprecate `view_type.attributes_type` in favor of `AttributesType[view_type]` [#149](https://github.com/redballoonsecurity/ofrak/pull/149)
```

Comment:
Nit -- technically "deprecated" is "soon-to-be removed features". Was `view_type.attributes_type` deprecated or removed?

If removed:
```suggestion
- Remove `view_type.attributes_type` in favor of `AttributesType[view_type]` [#149](https://github.com/redballoonsecurity/ofrak/pull/149)
```

Reply:
And maybe then put in the "Deprecated" or "Removed" section?

Reply:
It was deprecated